test/api/v3/integration/user/POST-user_class_cast_spellId.test.js

```diff
@@ -187,6 +187,30 @@ describe('POST /user/class/cast/:spellId', () => {
     expect(group.chat[0].uuid).to.equal('system');
   });
 
+  it('Etheal Surge does not recover mp of other mages', async () => {
```

Comment:
typo: Ethereal

Reply:
Thank you for noticing it! I'll fix it right now.